ncm-download/src/main/pan/components/download/schema.pan

```diff
@@ -41,6 +41,13 @@ type component_download_file = {
     "timeout" ? long
     @{allow older remote file}
     "allow_older" ? boolean
+    @{An dict with foreach daemon the CAF::Service action to take
```

Comment:
How about: 

```
A dict of daemonname => caf_serviceaction indicating what actions
should be applied to daemonname if the file changes. This allows,
for example, the restarting of a daemon upon file change.
Even if multiple services are associated to the same daemon, each action
for the daemon will be taken at most once.
If multiple actions are to be taken for the same daemon, all actions
will be taken (no attempt to optimize is made).}
```

Reply:
ok, i''l also change it in metaconfig when i redo #1341 on top of #1451